{"uuid":"fdc69408-0587-47b4-a4bf-b9cb6fed8454","historyId":"767ac3d32702723b2dda5b7f9ebd55ee","fullName":"com.giva.qa.testcases.LoginPageTest.LoginWithMultipleUser","labels":[{"name":"package","value":"com.giva.qa.testcases.LoginPageTest"},{"name":"testClass","value":"com.giva.qa.testcases.LoginPageTest"},{"name":"testMethod","value":"LoginWithMultipleUser"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"GivaTestAutomation_Trial"},{"name":"subSuite","value":"com.giva.qa.testcases.LoginPageTest"},{"name":"host","value":"Tsukki"},{"name":"thread","value":"16520@Tsukki.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"LoginWithMultipleUser","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.NullPointerException: Cannot invoke \"org.apache.poi.ss.usermodel.Sheet.getLastRowNum()\" because \"com.giva.qa.util.TestUtil.sheet\" is null","trace":"java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke \"org.apache.poi.ss.usermodel.Sheet.getLastRowNum()\" because \"com.giva.qa.util.TestUtil.sheet\" is null\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:803)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:728)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:924)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.NullPointerException: Cannot invoke \"org.apache.poi.ss.usermodel.Sheet.getLastRowNum()\" because \"com.giva.qa.util.TestUtil.sheet\" is null\r\n\tat com.giva.qa.util.TestUtil.getTestData(TestUtil.java:50)\r\n\tat com.giva.qa.testcases.LoginPageTest.getGivaTestData(LoginPageTest.java:39)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733561070778,"stop":1733561070780}